docs/content/guides/dagster/asset-versioning-and-caching.mdx

```diff
@@ -226,24 +234,26 @@ def multiplied_number(versioned_number):
     return versioned_number * 2
 ```
 
-Once again, both assets are now marked stale. Dagster doesn't know that `v5` of the versioned number will return the same value as `v4`, as it only knows about code versions and data versions.
+Once again, both assets have labels to indicate the change in code version. Dagster doesn't know that `v5` of the versioned number will return the same value as `v4`, as it only knows about code versions and data versions.
 
 Let's see what happens if only `versioned_number` is materialized. Select it in the asset graph and click **Materialize selected**. The sidebar shows the latest materialization now has a code_version of `v5`, and the data version is again `20`:
 
+<!-- ![Manual data versions 2](/images/guides/asset-versioning-and-caching/manual-data-versions-2.png) -->
+
 <Image
 alt="Manual data versions 2"
 src="/images/guides/asset-versioning-and-caching/manual-data-versions-2.png"
 width={2662}
-height={1748}
+height={1618}
 />
 
-Notice that `multiplied_number` is no longer stale, even though we didn't materialize it! Here's what happened: `versioned_number` is no longer stale, so `multiplied_number` is no longer automatically stale. Dagster then compared the data version of `versioned_number` last used to materialize `multiplied_number` to the current data version of `versioned_number`. Since this comparison shows that the data version of `versioned_number` hasn't changed, Dagster knows that `multiplied_number` isn't stale.
+Notice that `multiplied_number` is no longer has a label, even though we didn't materialize it! Here's what happened: the new materialization of `versioned_number` with the explicitly-supplied data version supercedes the code version of `versioned_number`. Dagster then compared the data version of `versioned_number` last used to materialize `multiplied_number` to the current data version of `versioned_number`. Since this comparison shows that the data version of `versioned_number` hasn't changed, Dagster knows that the change to the code version of `versioned_number` doesn't affecet `multiplied_number`.
```

Comment:
s/explicitly-supplied/explicitly supplied/

adverb + adjective not hyphenated

Reply:
"is no longer has a label"